contracts/DADBridgeWrapper.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: SEE LICENSE IN LICENSE
+pragma solidity 0.8.9;
+
+import { AccessControl } from "@openzeppelin/contracts/access/AccessControl.sol";
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+contract DADBridgeWrapper is AccessControl {
```

Comment:
let's also add the ability to pause/unpause minting (same permission), using OZ's pausable contract

Reply:
Done.